### ISSUE TITLE
[Feature]: prefer-web-first-assertions should warn on isVisible/textContent/innerText,isDisabled, etc.

### ISSUE THREAD

**Issue by mxschmitt** [playwright-community/eslint-plugin-playwright#186]
As a user I saw the following often in the wild:

```ts
const myValue = await page.locator('').isVisible();
expect(myValue).toBe('foobar');
```

would be great if we could catch that and maybe allow it in toPass in the ideal world.

**Comment by mskelton** [comment]
@mxschmitt From your example, I don't see why this wouldn't work using `toBeVisible`?

**Comment by mxschmitt** [comment]
Yes in this scenario toBeVisible() should just work! Don't re-call what I meant with the toPass back then.

**Comment by freddadalen** [comment]
It would be awesome to have a lint rule that warns about usages like `locator.textContent()` and instead recommends: `expect(locator).toHaveText()`

**Comment by mskelton** [comment]
@freddadalen That use case is what https://github.com/playwright-community/eslint-plugin-playwright/blob/main/docs/rules/prefer-web-first-assertions.md already supports. We currently don't support variable references which is what this issue is tracking.

**Comment by freddadalen** [comment]
@mskelton okey great! 😀
